nekoyume/Assets/_Scripts/Helper/ShortcutHelper.cs

```diff
@@ -10,13 +10,24 @@
 using Nekoyume.TableData;
 using Nekoyume.UI;
 using Nekoyume.UI.Module;
+using static Nekoyume.Helper.ShortcutHelper;
 
 namespace Nekoyume.Helper
 {
     public static class ShortcutHelper
     {
         private const int MaxCountOfAcquisitionStages = 2;
 
+        public enum PlaceType
+        {
+            Stage,
+            Shop = 3,
+            Arena = 4,
+            Quest = 5,
+            Staking = 7,
+            EventDungeonStage = 8,
+        }
```

Comment:
Where are the numbers 1 and 2?

Reply:
Previously too, 1 and 2 were not used, but I think we need further explanation. I'll add comments.